### ISSUE TITLE
Discussion: Decentralized Personalized Advertisements

### ISSUE THREAD

**Issue by vasa-develop** [simpleaswater/rfcs#19]
**What new topic/thread/tutorial are you proposing? Please describe.**
What are the possible ways to implement decentralized personal Ads? 
What projects are working on this?

**Comment by 0zAND1z** [comment]
Here is a privacy-first approach to Ad targeting based on local machine learning from Brave Browser: https://brave.com/brave-ads-waitlist/

**Comment by vasa-develop** [comment]
A rather inefficient way to do this can be this:

Using your browser activity, the browser can figure out your preferences(this maybe be hard to do on the low-end devices). 
And there can be a decentralized pub-sub of advertisements in which the advertisers can publish ads to specific keywords(keys for publishing part) and if your preferences match with some set of keywords, then your browser can subscribe to those keywords. 

@0zAND1z what do you think?

**Comment by vasa-develop** [comment]
At least there can be a POC based on this.

**Comment by 0zAND1z** [comment]
Yes, this could work at the edge level with ZERO centralization.
